iris/pom.xml

```diff
@@ -51,7 +52,12 @@
 		<dependency>
 			<groupId>org.opentestsystem.delivery</groupId>
 			<artifactId>item-renderer</artifactId>
-			<version>3.0.0-SNAPSHOT</version>
+			<version>${item-renderer.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.opentestsystem.delivery</groupId>
+			<artifactId>resource-bundler</artifactId>
+			<version>${item-renderer.version}</version>
```

Comment:
This is the only direct reference to ResourceBundler in this project.  I've updated it to refer to the artifact produced by the TDS_ItemRenderer project.